integration-tests/itest/deploy_test.go

```diff
@@ -97,14 +111,6 @@ var _ = Describe("Application deployment", func() {
 			close(done)
 		}, LongTimeout)
 
```

Comment:
unnecessary double-check as all Pods have to be in running state to allow the App to reach "running" state

Reply:
yeah maybe this was coded differently before, now it's just duplicated.